### PR TITLE
Add postmortem about the docs.rs outage

### PR DESCRIPTION
**Pull request by pietroalbini**
r? @Mark-Simulacrum

### CODE REVIEW

**Comment by pietroalbini** [comment]
Everyone involved with the incident response already reviewed the document.

File: posts/inside-rust/2019-10-21-docsrs-outage-postmortem.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+[ashleygwilliams]: https://github.com/ashleygwilliams
+[QuietMisdreavus]: https://github.com/QuietMisdreavus
+[Mark-Simulacrum]: https://github.com/Mark-Simulacrum
+[pietroalbini]: https://github.com/pietroalbini
+
+## Action items
+
+* Update the docs.rs source code to cleanup the offending directory
+  automatically.
+* Add alerts when the available disk space on a server reaches 10%.
```
Comment (lzutao) [inline posts/inside-rust/2019-10-21-docsrs-outage-postmortem.md]:
```suggestion
* Add alerts when the available disk space on a server below 10%.
```

Comment (pietroalbini) [reply posts/inside-rust/2019-10-21-docsrs-outage-postmortem.md]: *is below

**Comment by pietroalbini** [comment]
(just r=me, wait for me to merge it, thanks!)

**Comment by pietroalbini** [comment]
Discussed this at the docs.rs postmortem meeting, merging it.

**Pull request merged**